src/main/java/adrian/katana/code/VowelCount.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2018 Jala Foundation.
+ * 2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+package adrian.katana.code;
+/**
+ * Class description
+ *
+ * @author Adrian Rojas - AWT-[01].
+ * @version 1.0.
+ */
+
+/*
+Return the number (count) of vowels in the given string.
+We will consider a, e, i, o, and u as vowels for this Kata.
+The input string will only consist of lower case letters and/or spaces.
+* */
+public class VowelCount {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)